src/runtime/c_runtime_api.cc

```diff
@@ -95,7 +96,8 @@ DeviceAPI* DeviceAPI::Get(TVMContext ctx, bool allow_missing) {
 }
 
 void* DeviceAPI::AllocWorkspace(TVMContext ctx, size_t size) {
-  return AllocDataSpace(ctx, size, kTempAllocaAlignment);
+  TVMType type_hint = {.code = kDLUInt, .bits = 8, .lanes = 1};
```

Comment:
designated initializer is supported in C99 but not c++11 (need c++20), let us not do this for now

Reply:
Changed to `TVMType type_hint{kDLUInt, 8, 1}`.